src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceAfterCheck.java

```diff
@@ -91,14 +91,111 @@
  * <pre>
  * &lt;module name=&quot;NoWhitespaceAfter&quot;/&gt;
  * </pre>
- * <p>To configure the check to forbid linebreaks after a DOT token:
+ *
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * class Test2 {
+ *   public void function1() {}
+ *   public void function2() {}
+ * }
+ *
+ * class Test extends Test2 {
+ *   int p, q, r;
+ *   &#064; Override // violation, whitespace after '@' is not allowed
+ *   public void function1() {}
+ *
+ *   &#064;
+ *   Override // OK, default value of 'allowLineBreaks' is true
+ *   public void function2() {}
+ *
+ *   public void examples(int p) {
+ *     this. p = 5; // violation, whitespace after '.' is not allowed
+ *     this.q = 10; // OK
+ *     this.
+ *     r = 15; // OK, default value of 'allowLineBreaks' is true
+ *     int[] array1 = { 1,2,3,4}; // violation, whitespace after '{' is not allowed
+ *     int[] array2 = {1, 2, 3, 4 }; // OK
+ *     int[][] array3 = {{ 1,2}, {3,4}}; // violation, whitespace after '{' is not allowed
+ *     int[] array4 = {
+ *     1,
+ *     2
+ *     }; // OK, default value of 'allowLineBreaks' is true
+ *     int a=0, b;
+ *     ++ a; // violation, whitespace after '++' is not allowed
+ *     ++a; // OK
+ *     -- a; // violation, whitespace after '--' is not allowed
+ *     --a; // OK
+ *     b = - a; // violation, whitespace after '-' is not allowed
+ *     b = + a; // violation, whitespace after '+' is not allowed
+ *     b = -a; // OK
+ *     b = ~ a; // violation, whitespace after '~' is not allowed
+ *     b = ~a; // OK
+ *     boolean c = false, d;
+ *     d = ! c; // violation, whitespace after '!' is not allowed
+ *     d = !c; // OK
+ *   }
+ * }
+ * </pre>
+ *
+ * <p>
+ * To configure the check to forbid line breaks after DOT token:
  * </p>
  * <pre>
  * &lt;module name=&quot;NoWhitespaceAfter&quot;&gt;
  *   &lt;property name=&quot;tokens&quot; value=&quot;DOT&quot;/&gt;
  *   &lt;property name=&quot;allowLineBreaks&quot; value=&quot;false&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ *
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * class Test{
+ *   int a, b;
+ *   Test(int a, int b) {
+ *     this.
+ *         a = a; // violation, linebreak after '.' is not allowed
+ *     this.b = b; // OK
+ *   }
+ *
+ *   public void function() {}
+ *
+ *   public static void main(String[] args) {
```

Comment:
Please make code block same for all configs.
To show user how behavior on the same code is changing if you change properties of Check..

Reply:
done

Reply:
Not done